man.rst

```diff
@@ -581,6 +604,88 @@ from. These are the git config keys used:
 [1] https://developer.github.com/v3/pulls/#get-a-single-pull-request
 
 
+EXAMPLES
+========
```

Comment:
Nice! I'm not sure if you saw we have a bunch of examples in the README too. Not sure what to do with this, because it is definitely nice to have example both in the "home page" and in the man, and AFAIK there is no easy way to include them in both without duplicating them. So feel free to chose what makes more sense to you, but it might be good to have some sort of consistency between both set of examples. Not sure if you looked at the examples in the README.